yt_dlp/postprocessor/ffmpeg.py

```diff
@@ -1173,3 +1186,160 @@ def run(self, info):
             'ext': ie_copy['ext'],
         }]
         return files_to_delete, info
+
+
+class FFmpegProgressTracker:
+
+    def __init__(self, info_dict: dict, ffmpeg_args: List, ffmpeg_proc: Popen, hook_progress: Callable, running_status_value: str, action_bytes_key: str):
```

Comment:
why not actually needed rn, pass the `YoutubeDL` instance to the initializer as `self.ydl`. Otherwise we'll be stuck if we need to access any of it's functions/params in future